solidity/second_round/problem_one/instructions.md

```diff
@@ -0,0 +1,47 @@
+# Solidity Hiring Exercise #1
+
+```
+TIMEBOX:    2-4 hours max.
+LANGUAGES:  Solidity, Hardhat, Typescript
+WEB3 FRAMEWORKS: 
+TESTS:      Anchor tests
```

Comment:
what do you mean by anchor tests?

Reply:
oops that was a typo -- should be hardhat tests